common/pom.xml

```diff
@@ -104,5 +104,10 @@
             <groupId>io.github.aliyunmq</groupId>
             <artifactId>rocketmq-logback-classic</artifactId>
         </dependency>
+        <!-- rocksdb -->
+        <dependency>
+            <groupId>io.github.aliyunmq</groupId>
```

Comment:
Why rely on a commercial version? How to ensure the stability of this version?

Reply:
It's still open source, but we need a new compactionFilter for ConsumeQueue. If you're interested, see how flink uses rocksdb.

Reply:
@fujian-zfj  Your purpose of building custom rocksdb is to define a compaction filter. What a custom compaction filter expects is native function. Will this way be better alternative:
1. Use official rocksdb-java jar;
2. Build a shared library(dynamically linked with rocksdb shared library) for expected custom compaction filters;
3. When configuring RocksDB, pass the native function to it, because it does not matter which shared library the compaction filter stays in;
As a matter of fact, this is how the native programming languages use rocksdb in case of dynamic link.